.circleci/config.yml

```diff
@@ -506,7 +506,7 @@ jobs:
       - run: mkdir -p $TEST_RESULTS
       - run-acceptance-tests:
           failfast: true
-          additional-flags: -use-kind -kubecontext="kind-dc1" -secondary-kubecontext="kind-dc2"
+          additional-flags: -use-kind -kubecontext="kind-dc1" -secondary-kubecontext="kind-dc2" -helm-chart-version="0.47.1"
```

Comment:
testing in CI, will remove later.

Reply:
is there any way we can configure circle to git checkout a certain tag instead?

Reply:
This line itself will be reverted, and it will be internally using the chart associated with the commit that CI is running against.
For the nightly acceptance tests they will specify the above though. I guess we *could* somehow make it check things out in a certain way but its already hard coded everywhere such as the consul versions, envoy versions, etc. Thoughts?